### DISCUSSION TITLE
"Reader view" userscript equivalent for Min?

### DISCUSSION THREAD

**Discussion by dajare** [minbrowser/min#1741] [Q&A] [answered]
I've been enjoying using Min, but the more I use it, the more I find myself wanting/needing the ["Readerview" extension][1], available for Chrome, Firefox, Opera, and Edge.

Roughly, it targets the main text block, and gives an uncluttered, easy-to-read (as configured by the reader) version of the page. "[Readable][2]" is a bookmarklet service that does much the same thing—but there are many sites it seems not to operate on, and I'm unclear about the status of [bookmarklets with Min][3].

Min does not support these extensions, of course, but is there a "userscript" that can at least get me most of the way there? I live in hope!

[1]: https://add0n.com/chrome-reader-view.html
[2]: http://readable.tastefulwords.com/
[3]: https://github.com/minbrowser/min/issues/208

**Comment by PalmerAL** [comment, marked as answer]
This is already built in to Min, using the same library that the addon uses. On any page that's detected as an "article", there's a book icon you can click on to get the reader view:
<img width="242" alt="Screen Shot 2021-09-16 at 2 45 39 PM" src="https://user-images.githubusercontent.com/10314059/133676205-a059eb0b-d08a-4584-a260-bbb935332b64.png">
(example: https://www.nytimes.com/2021/09/16/us/school-shortages-bus-drivers-workers.html)

The detection of what's an "article" isn't always right, but you can press shift+cmd+r to override it and get reader view on any page.

**Reply by dajare** [reply]
Ahh, brilliant! I had not spotted that. 😬 I don't see any "controls" other than overall "theme" (light, sepia, dark), but this meets my *need* already. I suppose adding some adjustments (font size, font even!, custom colours) would be *very* low priority enhancements.

Thanks, too, for the quick reply! 👍

**Reply by PalmerAL** [reply]
Yeah there aren't any other settings (but there probably should be). The standard page zoom controls still work there, but they're not very precise.

What other fonts would you want to use?

**Reply by dajare** [reply]
Sorry for the delay in reply, and thanks for the response.

> What other fonts would you want to use?

I expect this is fairly personal to readers/users. I'm pretty happy with `font-family: "Charter", "Georgia", serif;` as it now appears in readerContent.css. **If** there was a way of letting users set this to a font on their system (I would, e.g., probably default to Merriweather for this), that would be ideal. Others might prefer a sans option (not Arial, I expect!), of course.

The other three parameters it would be great (IMO) to expose as options would be:

- font size,
- column width, and
- row height.

That's very much in the category of "wouldn't it be nice if...."! Meanwhile, it's great to see a usable "reader view" baked in to Min. :+1:
